Week-1/Session-03/DevTools_HandsOn.md

```diff
@@ -0,0 +1,37 @@
+    ### Situation: You want to inspect the HTML and CSS code of a webpage.
+    ### Question: Which panel in Chrome DevTools should you use to inspect the DOM and CSS of a webpage?
+    > Elements panel
+### 
+    ### Situation: You suspect that a particular JavaScript function is causing errors on a webpage.
+    ### Question: Which panel in Chrome DevTools should you use to debug the JavaScript code on a webpage?
+    > Sources panel
+### 
+    ### Situation: You want to analyze the network traffic of a webpage and find out which requests are taking the longest time to load.
+    ### Question: Which panel in Chrome DevTools should you use to analyze the network requests and responses of a webpage?
+    > Network panel
+### 
+    ### Situation: You want to inspect and manipulate browser storage, such as cookies and local storage.
+    ### Question: Which panel in Chrome DevTools should you use to inspect and manipulate browser storage on a webpage?
+    > Application panel
+### 
+    ### Situation: You want to optimize the performance of a webpage and identify potential bottlenecks.
+    ### Question: Which panel in Chrome DevTools should you use to analyze the performance of a webpage and identify potential bottlenecks?
+    > Performance panel
+### 
+    ### Situation: You want to test the security of a webpage and check if it has any vulnerabilities.
+    ### Question: Which panel in Chrome DevTools should you use to analyze the security of a webpage and identify potential vulnerabilities?
+    > Security panel
+### 
+    ### Situation: You want to view and modify the CSS styles of a webpage in real-time.
+    ### Question: Which panel in Chrome DevTools should you use to view and modify the CSS styles of a webpage in real-time?
+    > Elements panel(sub tab Styles)
+### 
+    ### Situation: You want to test how a webpage looks and behaves on different screen sizes.
+    Question: Which panel in Chrome DevTools should you use to test the responsiveness of a webpage on different screen sizes?
+    > Toggle Device Toolbar
+
+
+# Instructions
+
+### Go to chrome dev tools and analyze every panel and answer the question
+### Go to panel and play around according to each situation mentioned
```

Comment:
looks good, where are rest of the files for session 3 assignment?

Reply:
These are the instructions to follow to give the answer to the above questions in the file.